content/docs/next-release/frontend-ui.md

```diff
@@ -92,6 +100,11 @@ Links can either be members of the `menu` Array, directly, or they can be groupe
 
 The `items` Array should contain one or more link configurations.
 
+### Search Input
+
+`search.maxLookback` allows to define the max look back interval that the ui will show in the select input.
```

Comment:
What are the valid values for this?

Reply:
Mmm, I'm not sure about that. I'll check

Reply:
`search.maxLookback` configures the maximum time before the present users can query for traces.

Field | Description
------|------------
label | The text displayed in the search form dropdown
value | The limit used to truncate the `lookbackOptions` in `SearchForm.js`, and the value submitted in the search query if the label is selected

Raw for above markdown:
```
Field | Description
------|------------
label | The text displayed in the search form dropdown
value | The limit used to truncate the `lookbackOptions` in `SearchForm.js`, and the value submitted in the search query if the label is selected
```

Reply:
I've just added this. Let me know for any changes.
Thanks a lot @everett980!